### DISCUSSION TITLE
Welcome Scene card: how does it work to show on/off state in scenes?

### DISCUSSION THREAD

**Discussion by mvereecken** [UI-Lovelace-Minimalist/UI#661] [Q&A] [answered]
In your example in the documentation, HOME seems to be ON, the rest of the scenes is OFF. How can I achieve this? If it's in the documentation, I probably don't understand it. Can you explain?
![image](https://user-images.githubusercontent.com/3824067/168377500-afc71c93-3907-43b1-8796-1b1f4997e46a.png)

**Comment by basbruss** [comment, marked as answer]
This behavior is controlled by the state of the entity. Scenes do not provide a real state, but a timestamp that indicates the last moment it was activated. As this is not a state like `on`, `playing`, `open`, it will always show as `off`

Tip: you could always set an `input_boolean` to control the scene with automation which sets the `input_boolean` to `off` if one of the lights changes states after selecting the scene.

**Reply by mvereecken** [reply]
Thanks, this was helpful!
